src/components/features/reviews-section/components/reviews-list.tsx

```
@@ -0,0 +1,90 @@
+    // Formatter
+    const format = await getFormatter();
+
+    // State
+    let data: ReviewsSuccessResponse | null = null;
+    let error = false;
+
+    try {
+        // Fetch reviews from API
+        data = (await fetchProductReviews(productId)) as ReviewsSuccessResponse;
+    } catch (err) {
+        console.error("Failed to fetch product reviews:", err);
```

ESLint

[error] Unexpected console statement.

(no-console)